LayerClient/LayerClient.py

```diff
@@ -1,4 +1,6 @@
 # -*- coding: utf-8 -*-
+from __future__ import absolute_import, unicode_literals
```

Comment:
Are you running this on python < 2.7? I think absolute import is the default now